cubeviz/controls/tests/test_slice_controller.py

```diff
@@ -353,10 +353,10 @@ def test_wavelength_ui(qtbot, cubeviz_layout, slice_index=1000):
 
     # Units and negative redshift
     wui.do_calculation(wavelength_redshift=-3, wavelength_units='um')
-    assert_wavelength_text(cubeviz_layout, '-1.1029')
+    assert_wavelength_text(cubeviz_layout, '-9.6250e-01')
```

Comment:
Why did the value change here?  it is different than the -1.1029 from previously.

Reply:
I can't say for certain, but I believe it's because previously, cubeviz would immediately update its current wavelength location based on the vertical line in specviz. Now, however, the initial position is read directly from cubeviz and used to set the vertical line instead.